ui/apps/platform/src/Containers/ComplianceEnhanced/ScanConfigs/CreateScanConfigPage.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement } from 'react';
+
+import { Breadcrumb, BreadcrumbItem, Divider, PageSection, Title } from '@patternfly/react-core';
+
+import { complianceEnhancedScanConfigsBasePath } from 'routePaths';
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import PageTitle from 'Components/PageTitle';
+
+import ScanConfigWizardForm from './Wizard/ScanConfigWizardForm';
+
+function ScanConfigPage(): ReactElement {
+    return (
+        <>
+            <PageTitle title="Create Scan Config" />
+            <PageSection variant="light" className="pf-u-py-md">
+                <Breadcrumb>
+                    <BreadcrumbItemLink to={complianceEnhancedScanConfigsBasePath}>
+                        Scan schedules
+                    </BreadcrumbItemLink>
+                    <BreadcrumbItem isActive>Create scan schedule</BreadcrumbItem>
+                </Breadcrumb>
+            </PageSection>
+            <Divider component="div" />
+            <PageSection variant="light" padding={{ default: 'noPadding' }}>
+                <Title headingLevel="h1" className="pf-u-py-lg pf-u-px-lg">
+                    Create Scan Schedule
+                </Title>
+            </PageSection>
+            <Divider component="div" />
+            <PageSection padding={{ default: 'noPadding' }} isCenterAligned>
```

Comment:
Seeing 3 `PageSection` elements, which is common to be fair, brings another possibility to mind:

```tsx
    return (
        <>
            <PageTitle title="Create scan schedule" />
            <PageSection component="div" variant="light" className="pf-u-pb-0">
                <Flex direction={{ default: 'column' }}>
                    <Breadcrumb>
                        <BreadcrumbItemLink to={complianceEnhancedScanConfigsBasePath}>
                            Scan schedules
                        </BreadcrumbItemLink>
                        <BreadcrumbItem isActive>Create scan schedule</BreadcrumbItem>
                    </Breadcrumb>
                    <Divider component="div" />
                    <Title headingLevel="h1">Create scan schedule</Title>
                    <Divider component="div" />
                </Flex>
            </PageSection>
            <PageSection
                variant="light"
                isFilled
                hasOverflowScroll
                padding={{ default: 'noPadding' }}
                className="pf-u-h-100"
            >
                <ScanConfigWizardForm />
            </PageSection>
        </>
    );
```

* Use of `Flex` to reduce `className` props:
    https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Administration/Events/AdministrationEventPage.tsx#L53-L64
* Additional `PageSection` properties for wizard from Policies:
    https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Policies/Wizard/PolicyWizard.tsx#L149-L155

![PageSection](https://github.com/stackrox/stackrox/assets/11862657/46b278e0-d8bb-48a4-9e19-8c127aaf891c)

Reply:
I don't disagree but my main concern is keeping things consistent with Vulnerabilities and using that as the standard. Vulnerability reporting didn't have padding on the dividers so I wanted to create the same look here.